components/CreateCard.js

```diff
@@ -43,6 +50,7 @@ export default function CreateCard({ onAddBook }) {
           name="isbn"
           id="isbn"
           placeholder="9780571200832"
+          // autoComplete="off"
```

Comment:
why is this part in a comment?

if you don't need it, then it is better to delete it